src/components/Quiz/quiz.js

```diff
@@ -78,7 +78,7 @@ const Quiz = ({ children, state, dispatch }) => {
 	return (
 		<QuizWrapper>
 			<Navbar />
-			<ButtonArrow handleClick={handleClick} content='back' />
+			<ButtonArrow handleClick={handleClick} content={quizIndex === 0 ? 'first' : 'back'} />
```

Comment:
I think this should be extracted to a function to maintain the render code clean.
I think something like `getButtonArrowContent` would work.

WDYT??

Reply:
Thanks! 
Yep, actually sounds good! give me a minute and lets see how looks like that.

Reply:
I updated the branch, and this pull request was automatically updated.

Let me give you a little brief about this work. I'm choose this project because I like the main idea about Carbon Footprint and actually want to work with React and Open source projects, this is my firts approach with real React js proyects, i'm come from other stacks. Thanks for your welcome, cheers and happy codding.

Reply:
Great! I haven't been coding in this project for a little while bc I was changing jobs but really soon I will start working on it again. So many things to do (like testing). So you can jump into improving whatever you want or I will posting really soon issues of what needs to be done.

I am glad to have another active contributor. :)

Reply:
@all-contributors please add @Usuario001 for Code, Bug reports.


.

Reply:
well... this didn't work... let me try doing it on an issue.